cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -309,6 +311,8 @@ $(document).ready(function () {
 
 			// clear interval
 			clearInterval(this.interval);
+
+			$(window).trigger('structureboard_hidden.sideframe');
```

Comment:
@qris Using this logic, shouldn't every event be thrown at $(window)? There's a reason why bubbling occurs. Having said this, the sideframe is actually an entirely separate document (frame) isn't it? In this case, it would seem to me that it should it indeed be sent to $(window).

@FinalAngel thoughts?